### PR TITLE
UI Improvement for different mobile browsers

### PR DESCRIPTION
**Pull request by Youngjun333**
Did some minor css changes to be more inline with real smartphones

### CODE REVIEW

File: source/styles/timer.css [hunk 1]
```diff
@@ -280,25 +242,14 @@
     }
 
     #break-container {
-        margin-top: -35px;
-        margin-bottom: -84px;
-    }
-
-    .info-button {
```
Comment (jinbest17) [inline source/styles/timer.css:287]: Good job deleting duplicate style

**Review by jinbest17 — APPROVED** [review]
Good job on fixing the responsive design and making sure ui is consistent for multiple screen sizes

File: source/styles/main.css [hunk 2]
```diff
@@ -181,6 +181,37 @@ aside {
     animation: break-animation 1s infinite;
 }
 
+/* positioning for pomo pots */
```
Comment (friidryce) [inline source/styles/main.css:184]: constructive comments and naming convention

**Review by friidryce — APPROVED** [review]
Good job!

**Pull request merged**